### ISSUE TITLE
Python handler for docstring Examples inserts `<br>`

### ISSUE THREAD

**Issue by timvink** [pawamoy/mkdocstrings#194]
**Describe the bug**

Docstrings with an example section are parsed but always inserted with an extra `<br>`:

> ![image](https://user-images.githubusercontent.com/5570380/101941428-41f5a080-3be8-11eb-8442-0a3f30b1fa4a.png)
![image](https://user-images.githubusercontent.com/5570380/101941486-589bf780-3be8-11eb-8b0c-5143de7dba5c.png)

(you can see this live at https://ing-bank.github.io/probatus/api/feature_elimination.html)

**To Reproduce**

```python
git clone git@github.com:ing-bank/probatus.git
cd probatus
mkdocs serve
# browse to API page, many classes have code examples
```

Relevant docstring bit:

https://github.com/ing-bank/probatus/blob/ed5ab8cff256d03955fefe200d5d1ae24634b68f/probatus/feature_elimination/feature_elimination.py#L40-L45

**Expected behavior**

I don't expect the `<br>` to be inserted. This is inserted regardless of using spaces in the docstring btw.

> ![image](https://user-images.githubusercontent.com/5570380/101941827-d4963f80-3be8-11eb-8f43-a864ce1f477a.png)

**Information (please complete the following information):**
- OS: macOS
- Browser: chrome
- `mkdocstrings` version: 0.13.6

**Comment by pawamoy** [comment]
Hello, thanks for the report :slightly_smiling_face: 

I think I got a fix if you want to patch your installed pytkdocs and see if it works better:

In pytkdocs/parsers/docstrings/google.py, line 432, at the end of `read_examples_section`:

```diff
-        return Section(Section.Type.EXAMPLES, sub_sections), i
+        if sub_sections:
+            return Section(Section.Type.EXAMPLES, sub_sections), i
+
+        self.error(f"Empty return section at line {start_index}")
+        return None, i
```

Note that you may get a warning about an empty "examples" section. This is because `Example:` is recognized as the beginning of such a section, but the contents below are not indented. Also note that doctests are now supported (you'll still have to indent them):

```
Examples:
    >>> some python code
    # etc.
```

**Comment by pawamoy** [comment]
Should be fixed in pytkdocs 0.10.1. Please reopen or create a new issue if the problem persists!

**Comment by timvink** [comment]
Thanks!!
